package.yaml

```diff
@@ -0,0 +1,70 @@
+name:                servant-elm-street
+version:             0.1.0.0
+github:              "githubuser/servant-elm-street"
+license:             BSD3
+author:              "Author name here"
+maintainer:          "example@example.com"
+copyright:           "2021 Author name here"
+
+extra-source-files:
+- README.md
+- ChangeLog.md
+
+# Metadata used when publishing your package
+# synopsis:            Short description of your package
+# category:            Web
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
+description:         Please see the README on GitHub at <https://github.com/githubuser/servant-elm-street#readme>
+
+dependencies:
+- base >= 4.7 && < 5
+
+library:
+  source-dirs: src
+
+dependencies:
+- servant-foreign
+- text
+- servant-elm
+- base >= 4.7 && < 5
+- aeson
+- casing
+- elm-street
+- filepath
+- servant
+- servant-server
+- wai
+- warp
+- servant-js
+- random
+- elm-bridge
+- wl-pprint-text
+- microlens
```

Comment:
A lot of these dependencies are only used by your example API that you're using to run your code, once you remove it, you should also delete them.

Reply:
You can at least move all the things that are only used by the example API to the `dependencies` section of the executable, since they're only used there.

Reply:
done